tools/ingest-json-to-grok/src/main/java/org/logstash/ingest/JsonToGrok.java

```diff
@@ -0,0 +1,41 @@
+package org.logstash.ingest;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import javax.script.Invocable;
+import javax.script.ScriptEngine;
+import javax.script.ScriptEngineManager;
+import javax.script.ScriptException;
+
+/**
+ * Ingest JSON DSL to Logstash Grok Transpiler.
+ */
+public final class JsonToGrok {
+
+    private JsonToGrok() {
+        // Utility Wrapper for JS Script.
+    }
+
+    public static void main(final String... args) throws ScriptException, NoSuchMethodException {
```

Comment:
I think it's great to have individual `main` entry points for each processor so we can test them individually. So +1 on keeping this even when we have a consolidated converter for all processors.